### PR TITLE
add `kdb setmeta` examples

### PR DESCRIPTION
**Pull request by omnidan**

### CODE REVIEW

File: doc/help/elektra-cascading.md [hunk 1]
```diff
@@ -32,6 +32,12 @@ are available (in the meta data of respective `spec`-keys):
     the `fallback`-keys will be searched
 - `default`: this value will be used if nothing else was found
 
+They can be used like this:
+
+```
+$ kdb set /overrides/test "example override"
+$ sudo kdb setmeta spec/test override/#0 /overrides/test
+```
```
Comment (markus2330) [inline doc/help/elektra-cascading.md]: Please use the same syntax as done for all other examples:
(space space, newline, then tab, then single `)

**Comment by markus2330** [comment]
Thank you! Please fix syntax as in other parts.

File: doc/help/elektra-spec.md [hunk 2]
```diff
@@ -49,7 +49,14 @@ are now available (in the meta data of respective `spec`-keys):
     the `fallback`-keys will be searched
 - `default`: this value will be used if nothing else was found
 
-E.g.
+They can be used like this:
+
+```
+$ kdb set /overrides/test "example override"
+$ sudo kdb setmeta spec/test override/#0 /overrides/test
```
Comment (markus2330) [inline doc/help/elektra-spec.md]: Thats not necessary, because longer example follows.

**Comment by markus2330** [comment]
Other syntax only in kdb-setmeta.md, it seems like example is missing there.

**Comment by omnidan** [comment]
All issues should be fixed now.

**Comment by markus2330** [comment]
thank you

**Pull request merged**